backend/user-service/src/main/java/hu/me/iit/malus/thesis/user/service/impl/UserServiceImpl.java

```diff
@@ -240,19 +155,20 @@ public void saveCourseCreation(String teacherEmail, Long courseId) {
      */
     @Transactional
     @Override
-    public void saveCourseAssign(String studentEmail, Long courseId) {
-        Optional<Student> optionalStudent = studentRepository.findLockByEmail(studentEmail);
-        if (optionalStudent.isPresent()) {
+    public void assignStudentsToCourse(Long courseId, List<String> studentEmails) {
```

Comment:
van email uService. Nem hasznaljuk azt?

Reply:
az emailek a felhasználók primary keyei is egyben, szóval ez a user-service hastásköre. itt csak hozzárendeli a usereket a courseokhoz és küld emaileket is, azt már az email servicen keresztül